src/Files.App/Actions/FileSystem/CutItemAction.cs

```diff
@@ -31,11 +26,10 @@ public CutItemAction()
 			context.PropertyChanged += Context_PropertyChanged;
 		}
 
-		public Task ExecuteAsync()
+		public async Task ExecuteAsync()
 		{
 			if (context.ShellPage is not null)
-				UIFilesystemHelpers.CutItem(context.ShellPage);
-			return Task.CompletedTask;
+				await UIFilesystemHelpers.CutItem(context.ShellPage);
```

Comment:
You can remove `async` and then use something like:
```
    return context.ShellPage is not null
        ? UIFilesystemHelpers.CutItem(context.ShellPage)
        : Task.CompletedTask;
```

Reply:
CutItem is async operation

Reply:
Done